### PR TITLE
fix returned id components

### PR DESCRIPTION
**Pull request by migoi**
## Description

fix returned values

## Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [ ] My code is written according to the guidelines found [here] (https://yetiforce.com/en/developer-documentation/standards/167-how-to-create-php-files.html).
- [ ] I have signed the Contributor Licence Agreement between me and YetiForce.

### CODE REVIEW

File: include/Webservices/Utils.php [hunk 1]
```diff
@@ -442,6 +444,7 @@ function vtws_addWebserviceOperationParam($operationId, $paramName, $paramType,
  */
 function vtws_getModuleHandlerFromName($name, $user)
 {
+	global $log;
```
Comment (mariuszkrzaczkowski) [inline include/Webservices/Utils.php]: Global variable or function should never be used
https://insight.sensiolabs.com/projects/189f9068-d777-44a7-9164-9242e81df88c/analyses/163#rule-24-038

File: include/Webservices/Utils.php [hunk 2]
```diff
@@ -123,7 +123,9 @@ function vtws_getUserWebservicesGroups($tabId, $user)
 
 function vtws_getIdComponents($elementid)
 {
-	return explode("x", $elementid);
+	if (strpos($elementid, 'x') !== false) {
+		return explode("x", $elementid);
+	}
 }
```
Comment (mariuszkrzaczkowski) [inline include/Webservices/Utils.php]: If there is no x should return $elementid

File: include/Webservices/Utils.php (hunk 1 above)
Comment (migoi) [inline include/Webservices/Utils.php]: ok, removed

File: include/Webservices/Utils.php (hunk 2 above)
Comment (migoi) [inline include/Webservices/Utils.php]: in this case other side was expecting an array. However: is it still needed because of API?

**Pull request merged**